tutorials/darcy_thermo_mech/doc/content/workshop/problem/step10.md

```diff
@@ -75,3 +76,4 @@ cd problems
 ## Step 10: Multi-scale Results
 
 !media darcy_thermo_mech/step10_result.mp4
+       alt=The evolution of the macro-scale temperature and the phase of the material at the micro-scale, produced by the multi-scale simulation above.
```

Comment:
```suggestion
       alt=The evolution of the macro-scale temperature and the phase of the material at the micro-scale, produced by the multiscale simulation above.
```

We tend to remove the hyphen for "multi-scale" in our documentation and papers, so I'll keep that consistent here.